File: api/app/schemas/review.py
"""
복습 관련 Pydantic 스키마
"""
from pydantic import BaseModel
from typing import Optional


class ReviewQueueItem(BaseModel):
    unit_id: str
    lesson_id: Optional[str]
    reason: str  # "WRONG", "WRONG_REPEATED"
    priority: int


class ReviewComplete(BaseModel):
    user_id: str
    unit_id: str

File: apps/web/src/types/review.ts
/**
 * 복습 관련 타입 정의
 */
export interface ReviewQueueItem {
  unit_id: string;
  lesson_id?: string;
  reason: string;
  priority: number;
}

export interface ReviewComplete {
  user_id: string;
  unit_id: string;
}
